experimental/traffic-portal/src/app/core/cdns/cdn-detail/cdn-detail.component.html

```diff
@@ -0,0 +1,41 @@
+<!--
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<mat-card>
+	<tp-loading *ngIf="!cdn"></tp-loading>
+	<form ngNativeValidate (ngSubmit)="submit($event)" *ngIf="cdn">
+		<mat-card-content>
+			<mat-form-field>
+				<mat-label>Name</mat-label>
+				<input matInput type="text" name="name" required [(ngModel)]="cdn.name">
+			</mat-form-field>
+			<mat-form-field *ngIf="!new">
+				<mat-label>ID</mat-label>
+				<input matInput type="text" name="id" disabled readonly [defaultValue]="cdn.id">
+			</mat-form-field>
+			<mat-form-field *ngIf="!new">
+				<mat-label>Last Updated</mat-label>
+				<input matInput type="text" name="lastUpdated" disabled readonly [defaultValue]="cdn.lastUpdated">
+			</mat-form-field>
+			<mat-form-field>
+				<mat-label>Domain</mat-label>
+				<input matInput type="text" name="domainName" required [(ngModel)]="cdn.domainName">
```

Comment:
This allows invalid domain names to be entered.

Reply:
Added a `pattern` for CDN domain name in 3528126e87.